command/utils.go

```diff
@@ -131,7 +131,50 @@ func PrintInstances(wrt io.Writer, instances []*manager.Instance) error {
 	}
 	err := w.Flush()
 	return err
+}
+
+// PrintDocuments writes the output from ListDocuments.
+func PrintDocuments(wrt io.Writer, documents []*manager.DocumentIdentifier) error {
+	w := tabwriter.NewWriter(wrt, 0, 8, 1, ' ', 0)
+	header := []string{
+		"Name",
+		"Owner",
+		"Document Version",
+		"Document Format",
+		"Document Type",
+		"Schema Version",
+		"Target Type",
+	}
 
+	if _, err := fmt.Fprintln(w, strings.Join(header, "\t|\t")); err != nil {
+		return err
+	}
+	for _, document := range documents {
+		if _, err := fmt.Fprintln(w, document.TabString()); err != nil {
+			return err
+		}
+	}
+	err := w.Flush()
+	return err
+
+}
+
+// PrintDocumentDescription writes the output from DescribeDocument.
+func PrintDocumentDescription(wrt io.Writer, document *manager.DocumentDescription) error {
+	fmt.Fprintf(wrt, "Name: %s\n", document.Name)
```

Comment:
I'm thinking that this could perhaps be the `String()` method for `DocumentDescription`? Then it can be passed directly to e.g. `fmt.Println`?

Reply:
That would be possible, I'm not done with the output yet and will have a look at this.